android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -139,6 +157,14 @@ open class Amplitude(
          */
         internal const val DUMMY_EXIT_FOREGROUND_EVENT = "dummy_exit_foreground"
     }
+
+    override fun onNetworkAvailable() {
```

Comment:
@Mercy811 to make the question a bit clear, not quire sure where this method is used and why we need this.

Reply:
This is not used anywhere but I kept it here only for scalability in case it will be used in the future. In `networkChangeHandler`, `onNetworkAvailable` is left as blank.

Reply:
After implementing the [plugin architecture](https://github.com/amplitude/Amplitude-Kotlin/pull/171#discussion_r1439992857), this callback is used to change `configuraiton.isNetworkConnected`